packages/data/src/world/mm/snowhead_temple.yml

```diff
@@ -241,8 +249,8 @@
     "Snowhead Temple Center Level 3 Snow": "true"
     "Snowhead Temple Center Level 4": "(trick(MM_SHT_PILLARLESS) && (can_use_fire_arrows || has_bombs)) || (short_hook_anywhere && (has_bombs || can_use_fire_arrows)) || hookshot_anywhere"
   locations:
-    "Snowhead Temple SF Dinolfos 1": "soul_lizalfos_dinalfos"
-    "Snowhead Temple SF Dinolfos 2": "soul_lizalfos_dinalfos"
+    "Snowhead Temple SF Dinolfos 1": "soul_lizalfos_dinalfos && (can_fight || has_arrows)"
+    "Snowhead Temple SF Dinolfos 2": "soul_lizalfos_dinalfos && (can_fight || has_arrows)"
```

Comment:
We have Deku Bubbles in logic for them on the Moon too, although that one is only one.

Reply:
I used the same logic as for the dinolfos in woodfall temple. Should they all change to account for deku bubble?

Reply:
WFT one could. but since there's two in this room, it'd probably be too risky of getting burned.